Kubernetes/windows/fluent/fluent.conf

```diff
@@ -0,0 +1,80 @@
+# Things to check:
+# Time of generation and time of ingetion (look at the difference)
+# Can windows update happen? it can break fluent-bit/fluentd possibly
+# Persistance (so that logs are continuous) and write position file to host file system
+# when log entry is json, does it work?(proably needs a setting) -> #     Decode_Field_As   escaped    log -> find similar setting in fluentd?(parse json directive in input plugin)
+
+<source>
+  type heartbeat_request
+  run_interval 30m
+  @log_level info
+</source>
+
+<source>
+  @type tail
+  path /var/log/containers/*.log
+  pos_file /var/opt/microsoft/fluent/fluentd-containers.log.pos
+  tag oms.container.log.la
+  @log_level trace
+  path_key tailed_path
+  read_from_head true
```

Comment:
remove read_from_head true

Reply:
limit_recently_modified 5m + read_from_head true is the correct combination (agent upgrae/container restart scenario)